lib/rules/no-empty-pattern.js

```diff
@@ -10,6 +10,8 @@
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
I'd consider this to be a `problem` since reported problems are usually bugs.

Reply:
Agreed